.github/workflows/tests_coverage_deploy.yml

```
@@ -82,254 +76,80 @@ jobs:
-            echo "Deployment completed"
-          ENDSSH
\ No newline at end of file
+    - uses: webfactory/ssh-agent@v0.9.0
+      with:
+        ssh-private-key: ${{ secrets.EC2_PRIVATE_KEY }}
+
+    - name: Deploy to EC2
+      run: |
+        ssh -o StrictHostKeyChecking=no ${{ secrets.EC2_USERNAME }}@${{ secrets.EC2_HOST }} << 'ENDSSH'
+          set -e
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)